department/department_app/models/department.py

```diff
@@ -0,0 +1,13 @@
+"""models"""
+from django.db import models
+
+
+class Department(models.Model):
+    """Department"""
+    department = models.CharField(max_length=25)
+    dumber_of_employees = models.IntegerField(null=True)
+    the_average_salary = models.IntegerField(null=True)
```

Comment:
In accordance with the task, the average salary must be calculated automatically. Please remove this field and don't forget to generate a new migration script.

Reply:
fixed